pennylane/qnodes/passthru.py

```diff
@@ -79,7 +79,12 @@ def __init__(self, func, device, **kwargs):
         # make the device return the result in its native type
         kwargs = kwargs or {}
         kwargs.setdefault("use_native_type", True)
-        super().__init__(func, device, mutable=True, **kwargs)
+        kwargs.setdefault("mutable", True)
+
+        if not kwargs.get("mutable"):
+            raise ValueError("PassthruQNode does not support immutable mode.")
```

Comment:
How come?

Reply:
Oh, I'm not too sure 😆 All I know is that Ville explicitly hardcoded it as `mutable=True`, so I assumed there was a reason for it

Reply:
The reason looks to be in the module docstring:


> `PassthruQNode` does not have a Jacobian method, so it does not HAVE to use Variables or scalar linear indexing of input parameters.
> Two options:
> 1. Use Variables anyway, re-use most BaseQNode methods.
>    Problem: after evaluating the Variables, stacking sliced/indexed Tensors in Operation.parameters should somehow result in a Tensor, not an object array.
> 2. Do not use Variables, call the qfunc each time :meth:`PassthruQNode.evaluate` is called (always mutable).
>    Problem: tensornet_tf requires variable_deps?

It looks like Ville went for option (2), and forces construction to occur with every evaluation. Hence, the PassthruQnode can never be immutable by construction.